README.md

```diff
@@ -50,31 +50,25 @@ the central point of interaction for users:
   an update mechanism so there will only need to be a single store xdc app in
   the store bot chat. 
 
-## Setting Up the Bot
+## Setting up the bot
```

Comment:
We agreed to `Settung Up the Bot` as general heading styling. You should also align the other headings this way

Reply:
I will not do that, at least not in this PR, there are headers with long text and would look awful all capitalized etc